doc/extensions.rst

```diff
@@ -27,9 +27,14 @@ Connecting new machine learning libraries
 Content of the Library
 ~~~~~~~~~~~~~~~~~~~~~~
 
-To leverage support from the community and to tap in the potential of OpenML, interfacing
-with popular machine learning libraries is essential. However, the OpenML-Python team does
-not have the capacity to develop and maintain such interfaces on its own. For this, we
+To leverage support from the community and to tap in the potential of OpenML,
+interfacing with popular machine learning libraries is essential.
+The OpenML-Python package is capable of downloading meta-data and results (data,
+flows, runs), regardless of the library that was used to upload it.
+However, uploading flows and runs from a specific library, requires an
```

Comment:
Hm, I would say that the extensions simplify this process as one can go without them (https://openml.github.io/openml-python/main/examples/30_extended/custom_flow_.html#sphx-glr-examples-30-extended-custom-flow-py)